server/handlers/result_handlers.py

```diff
@@ -172,6 +183,22 @@ def __call__(self, result_batch: list[LSEBatchItem]):
         else:
             logger.error(f'No valid results to send to LSE for modelrun_id {self.modelrun_id}')
 
+        # Send failed predictions back to LSE
+        if error_batch:
+            error_batch = self.prepare_errors_payload(error_batch)
+            self.client.make_request(
```

Comment:
We need to start rewriting this to use label-studio SDK (cc @matt-bernstein )

Reply:
yep, doesn't have to be in this ticket but I made a general "stuff to deprecate in adala" ticket and added this to it https://humansignal.atlassian.net/browse/DIA-1337